signerv2/README.md

```diff
@@ -1,3 +1,60 @@
-# Signerv2
+# Signer v2
 
-TODO
\ No newline at end of file
+Signerv2 is a module for signing messages. It provides a simple and unified way to produce cryptographic signatures.
+Signers instantiated from this module is required to create some SDK transaction managers (see [`NewPrivateKeyWallet`](../chainio/clients/wallet/privatekey_wallet.go) and [`NewSimpleTxManager`](../chainio/txmgr/simple.go)/[`NewGeometricTxnManager`](../chainio/txmgr/geometric/geometric.go)).
+
+## Features
+
+- Sign messages using raw private keys
+- Sign messages using encrypted keystores
+- Sign messages using a remote signer (web3 or KMS)
+
+### Comparison to Old Signer
+
+In comparison to the old signer, Signerv2 offers:
+
+- New signing mechanisms
+- A simplified API for easier extension
+
+### Using SignerFromConfig
+
+SignerV2 introduces `SignerFromConfig`
```

Comment:
Might be nice to show and break down what the config contains here in the docs

Reply:
I added the suggestion to #440